src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let cost = days * 40;
+
+  if (days >= 7) {
+    cost -= 50;
+  } else if (days >= 3) {
+    cost -= 20;
+  }
+
+  return cost;
 }
 
 module.exports = calculateRentalCost;
+
+calculateRentalCost(25);
```

Comment:
Remove the function call at the end of the file as it is not necessary for the module export.